stable/rabbitmq-ha/templates/statefulset.yaml

```diff
@@ -1,26 +1,26 @@
 apiVersion: apps/v1beta1
 kind: StatefulSet
 metadata:
-  name: {{ template "rabbitmq-ha.fullname" . }}
+  name: {{ include "rabbitmq-ha.fullname" . }}
   labels:
-    app: {{ template "rabbitmq-ha.name" . }}
-    chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
-    release: {{ .Release.Name }}
-    heritage: {{ .Release.Service }}
+    app.kubernetes.io/name: {{ include "rabbitmq-ha.name" . }}
+    helm.sh/chart: {{ include "rabbitmq-ha.chart" . }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
 {{- if .Values.extraLabels }}
 {{ toYaml .Values.extraLabels | indent 4 }}
 {{- end }}
 spec:
   podManagementPolicy: {{ .Values.podManagementPolicy }}
-  serviceName: {{ template "rabbitmq-ha.fullname" . }}-discovery
+  serviceName: {{ include "rabbitmq-ha.fullname" . }}-discovery
   replicas: {{ .Values.replicaCount }}
   updateStrategy:
     type: {{ .Values.updateStrategy }}
   template:
     metadata:
       labels:
-        app: {{ template "rabbitmq-ha.name" . }}
-        release: {{ .Release.Name }}
+        app.kubernetes.io/name: {{ include "rabbitmq-ha.name" . }}
+        app.kubernetes.io/instance: {{ .Release.Name }}
```

Comment:
Changing pod labels breaks existing installations and does not allow for upgrades. I don't see the benefit of such a major incompatible change just to follow new labeling recommendations.

Reply:
It standard for now, so sooner or later it should be integrated into charts.
It can be new major version.